setup.cfg

```diff
@@ -20,3 +20,38 @@ universal = 1
 [metadata]
 license_file = LICENSE
 version = 40.8.0
+
+[setuptools]
+description= "Easily download, build, install, upgrade, and uninstall Python packages"
+author="Python Packaging Authority"
+author_email="distutils-sig@python.org"
+long_description=long_description
+long_description_content_type='text/x-rst; charset=UTF-8'
+keywords="CPAN PyPI distutils eggs package management"
+url="https://github.com/pypa/setuptools"
+project_urls= "Documentation": "https://setuptools.readthedocs.io/"
+src_root=None
+packages=setuptools.find_packages(exclude=['*.tests'])
```

Comment:
This is actually a function call in Python, in `setuptools` you annoyingly have to put it in two places, this line needs to be `packages = find:`

Then you create a new header called `[options.packages.find]`, and put 'exclude = *.tests` under it.